src/Microsoft.DotNet.Darc/DarcLib/LocalGitClient.cs

```diff
@@ -194,9 +197,22 @@ public async Task<string> AddRemoteIfMissingAsync(string repoPath, string repoUr
 
     public async Task UpdateRemoteAsync(string repoPath, string remoteName, CancellationToken cancellationToken = default)
     {
-        var result = await _processManager.ExecuteGit(repoPath, new[] { "remote", "update", remoteName }, cancellationToken: cancellationToken);
```

Comment:
were we not adding the authentication header before?

Reply:
Yeah, we were not